src/components/Navigation/Navigation.module.css

```diff
@@ -1,46 +1,33 @@
 .nav {
   background-color: var(--primary-background-color);
   display: flex;
-  width: 100%;
-  max-width: 1440px;
-  justify-content: space-between;
-  align-items: center;
-  margin: 0 auto;
-  padding: 0 81px 0 71px;
-  font-size: 1rem;
+  flex-direction: column;
+  width: 298px;
+  position: fixed;
+  z-index: 999;
+  right: 0;
+  font-size: 20px;
```

Comment:
a fonty nie mieliśmy przerabiać na rem ?

Reply:
Właśnie zrobiłem REM ale nie wiem czemu zmieniłem na px xd Poprawię to,